### PR TITLE
Feature/rel1 16/remove base crud controller

### PR DESCRIPTION
**Pull request by Caromeen**

### CODE REVIEW

File: src/main/java/org/feeder/api/authorization/client/ClientController.java [hunk 1]
```diff
@@ -1,28 +1,78 @@
...
+  }
+
+  @DeleteMapping(ID_PATH)
+  public ResponseEntity<ClientResponseVO> delete(@PathVariable final UUID id) {
+    getService().delete(id);
+    return ResponseEntity.status(NO_CONTENT)
+        .build();
+  }
+
   protected BaseCrudService<Client, ClientRequestVO, ClientResponseVO> getService() {
```
Comment (avarashen) [inline src/main/java/org/feeder/api/authorization/client/ClientController.java]: pls remove method

File: src/main/java/org/feeder/api/authorization/client/ClientController.java [hunk 2]
```diff
@@ -1,28 +1,78 @@
...
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 @RequiredArgsConstructor
 @RequestMapping(CLIENT_PATH)
-public class ClientController extends
-    BaseCrudController<Client, ClientRequestVO, ClientResponseVO> {
+public class ClientController {
 
+  protected static final String ID_PATH = "/{id}";
```
Comment (avarashen) [inline src/main/java/org/feeder/api/authorization/client/ClientController.java]: ID_PATH should be private

File: src/main/java/org/feeder/api/authorization/user/UserController.java [hunk 3]
```diff
@@ -1,27 +1,78 @@
...
+  }
+
+  @DeleteMapping(ID_PATH)
+  public ResponseEntity<UserResponseVO> delete(@PathVariable final UUID id) {
+    getService().delete(id);
+    return ResponseEntity.status(NO_CONTENT)
+        .build();
+  }
 
-  @Override
```
Comment (avarashen) [inline src/main/java/org/feeder/api/authorization/user/UserController.java:24]: pls remove method

File: src/main/java/org/feeder/api/authorization/user/UserController.java [hunk 4]
```diff
@@ -1,27 +1,78 @@
...
 @RestController
 @RequiredArgsConstructor
 @RequestMapping(USER_PATH)
-public class UserController extends BaseCrudController<User, UserRequestVO, UserResponseVO> {
+public class UserController {
 
   protected static final String USER_PATH = "/user";
-
   private final UserService service;
+  protected static final String ID_PATH = "/{id}";
```
Comment (avarashen) [inline src/main/java/org/feeder/api/authorization/user/UserController.java]: ID_PATH should be private and placed higher than user service

File: src/main/java/org/feeder/api/authorization/user/UserController.java [hunk 5]
```diff
@@ -1,27 +1,78 @@
...
+
+  @DeleteMapping(ID_PATH)
+  public ResponseEntity<UserResponseVO> delete(@PathVariable final UUID id) {
+    getService().delete(id);
+    return ResponseEntity.status(NO_CONTENT)
+        .build();
+  }
 
-  @Override
   protected BaseCrudService<User, UserRequestVO, UserResponseVO> getService() {
```
Comment (avarashen) [inline src/main/java/org/feeder/api/authorization/user/UserController.java]: pls remove method

**Review by texazcowboy — APPROVED** [review]

**Pull request merged**